packages/compose/src/utils/create-higher-order-component/index.ts

```diff
@@ -25,8 +25,8 @@ export type HigherOrderComponent< HOCProps extends Record< string, any > > = <
  * Given a function mapping a component to an enhanced component and modifier
```

Comment:
These changes were generated with `npm run lint-js:fix`. They are unrelated to the implementation details of the component being tested, or the tests.